app/models/user.rb

```diff
@@ -158,6 +159,51 @@ def self.authenticate(options)
     user
   end
 
+  aasm :column => :status, :no_direct_assignment => true do
```

Comment:
Since the user model file is already very long, [this aasm section](https://github.com/openstreetmap/openstreetmap-website/blob/1a11c4dc191d93b18fcf5aa917448c8cd6d2556b/app/models/user.rb#L162-L205) would be a good candidate to extract into a concern IMO. However, skimming over https://codeclimate.com/blog/7-ways-to-decompose-fat-activerecord-models/ again suggests that a Concern is also smelly, so maybe there is a better way to separate the large file(?).